src/util/ast.ts

```diff
@@ -73,13 +74,18 @@ export function convertObjectExpressionToValueDefinition(objectExpression: Acorn
       break
   }
 
-  if (!type) return
+  if (!type) {
+    return
+  } else {
+    loc = typeProperty?.value.loc
+  }
```

Comment:
I think you can remove the `let` on L65 and just use this here:

```suggestion
  if (!type) return

 let loc = typeProperty?.value.loc
```